Src/FluentAssertions/Collections/GenericDictionaryAssertions.cs

```diff
@@ -1275,5 +920,28 @@ public AndConstraint<TAssertions> NotContain(params KeyValuePair<TKey, TValue>[]
         /// Returns the type of the subject the assertion applies on.
         /// </summary>
         protected override string Identifier => "dictionary";
+
+        private static IEnumerable<TKey> GetKeys(TCollection collection) =>
```

Comment:
Do you still need these if you already have the helper extension methods?

Reply:
Yes.
Using e.g. `Subject.GetKeys()` from the helper class directly in an assertion body gives me `CS0411`

>	The type arguments for method 'DictionaryHelpers.GetKeys<TCollection, TKey, TValue>(TCollection)' cannot be inferred from the usage. 
> Try specifying the type arguments explicitly.

The purpose of all these methods in the bottom is to explicitly pass in the type arguments to the helper class, to avoid needing to specify them in the assertion bodies.